dojo/finding_group/signals.py

```diff
@@ -0,0 +1,31 @@
+from auditlog.models import LogEntry
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.dispatch import receiver
+from django.db.models.signals import post_delete
+from django.utils.translation import gettext as _
+from django.urls import reverse
+
+from dojo.models import Finding_Group
+from dojo.notifications.helper import create_notification
+
+
+@receiver(post_delete, sender=Finding_Group)
+def finding_group_post_delete(sender, instance, using, origin, **kwargs):
+    if instance == origin:
+        if settings.ENABLE_AUDITLOG:
+            le = LogEntry.objects.get(
+                    action=LogEntry.Action.DELETE,
+                    content_type=ContentType.objects.get(app_label='dojo', model='finding_group'),
+                    object_id=instance.id
+            )
+            description = _('The finding group "%(name)s" was deleted by %(user)s') % {
+                                'name': instance.name, 'user': le.actor}
```

Comment:
In this case, `le.actor` is properly populated with the username when a Finding Group is deleted in the UI.

![Screenshot 2024-04-09 at 20 27 29](https://github.com/DefectDojo/django-DefectDojo/assets/1749665/99b907c4-9698-4396-9222-2eadc9ed3c50)